SpatialGDK/Build/Programs/Improbable.Unreal.Scripts/Improbable.Unreal.Scripts.sln

```diff
@@ -37,4 +31,7 @@ Global
 	GlobalSection(SolutionProperties) = preSolution
 		HideSolutionNode = FALSE
 	EndGlobalSection
+	GlobalSection(ExtensibilityGlobals) = postSolution
```

Comment:
What is this for?

Reply:
Not sure added by vs2017 https://stackoverflow.com/questions/45861679/what-is-the-new-globalsection-in-a-vs2017-15-3-solution-file will see if I can remove it